requirements.txt

```diff
@@ -1,2 +1,2 @@
-Django>=1.4,<1.8
+Django>=1.4,<2.0
```

Comment:
The upcoming version is 1.10, not 2.0.
We could also make 1.7 the minimum version, other versions are really old, and Django 1.4 requires that `{% load url from future %}` you removed below. Could you do this change as well, please?